### ISSUE TITLE
module 'deeplabcut' has no attribute 'pose_estimation_tensorflow'

### ISSUE THREAD

**Issue by ghost** [DeepLabCut/DeepLabCut#1515]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Bug description

Similar to #1503 but I am not using Docker, I am using Anaconda (in Windows), and Python is already v3.8. I am using the current version of the provided environment files. 

### Operating System

Windows 10

### DeepLabCut version

I am not able to `import deeplabcut` so I can't be sure. Might be a good idea to include a txt file that plainly lists the version?

### DeepLabCut mode

multi animal

### Device type

cpu

### Steps To Reproduce

1. Install Anaconda
2. `git clone https://github.com/DeepLabCut/DeepLabCut.git`
3. `cd` into `conda-environments` folder and `conda env create -f DEEPLABCUT.yaml`
4. `conda activate DEEPLABCUT`
5. `ipython`
6. `import deeplabcut`

### Relevant log output

```shell
In [1]: import deeplabcut
2021-09-22 11:17:58.256281: W tensorflow/stream_executor/platform/default/dso_loader.cc:64] Could not load dynamic library 'cudart64_110.dll'; dlerror: cudart64_110.dll not found
2021-09-22 11:17:58.256652: I tensorflow/stream_executor/cuda/cudart_stub.cc:29] Ignore above cudart dlerror if you do not have a GPU set up on your machine.
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-1-cfa4f159dfc5> in <module>
----> 1 import deeplabcut

~\AppData\Local\Continuum\anaconda3\envs\dlc2\lib\site-packages\deeplabcut\__init__.py in <module>
     25
     26     mpl.use("WxAgg")
---> 27     from deeplabcut import generate_training_dataset
     28     from deeplabcut import refine_training_dataset
     29

~\AppData\Local\Continuum\anaconda3\envs\dlc2\lib\site-packages\deeplabcut\generate_training_dataset\__init__.py in <module>
     10
     11 from deeplabcut.generate_training_dataset.frame_extraction import *
---> 12 from deeplabcut.generate_training_dataset.trainingsetmanipulation import *
     13 from deeplabcut.generate_training_dataset.multiple_individuals_trainingsetmanipulation import *

~\AppData\Local\Continuum\anaconda3\envs\dlc2\lib\site-packages\deeplabcut\generate_training_dataset\trainingsetmanipulation.py in <module>
     21 import yaml
     22
---> 23 from deeplabcut.pose_estimation_tensorflow import training
     24 from deeplabcut.utils import (
     25     auxiliaryfunctions,

~\AppData\Local\Continuum\anaconda3\envs\dlc2\lib\site-packages\deeplabcut\pose_estimation_tensorflow\__init__.py in <module>
     13 from deeplabcut.pose_estimation_tensorflow.default_config import *
     14 from deeplabcut.pose_estimation_tensorflow.core.evaluate import *
---> 15 from deeplabcut.pose_estimation_tensorflow.core.train import *
     16 from deeplabcut.pose_estimation_tensorflow.core.test import *
     17 from deeplabcut.pose_estimation_tensorflow.export import export_model

~\AppData\Local\Continuum\anaconda3\envs\dlc2\lib\site-packages\deeplabcut\pose_estimation_tensorflow\core\train.py in <module>
     27     PoseDatasetFactory,
     28 )
---> 29 from deeplabcut.pose_estimation_tensorflow.nnets import PoseNetFactory
     30 from deeplabcut.pose_estimation_tensorflow.util.logging import setup_logging
     31

~\AppData\Local\Continuum\anaconda3\envs\dlc2\lib\site-packages\deeplabcut\pose_estimation_tensorflow\nnets\__init__.py in <module>
     13 """
     14 from .factory import PoseNetFactory
---> 15 from .efficientnet import PoseEfficientNet
     16 from .mobilenet import PoseMobileNet
     17 from .multi import PoseMultiNet

~\AppData\Local\Continuum\anaconda3\envs\dlc2\lib\site-packages\deeplabcut\pose_estimation_tensorflow\nnets\efficientnet.py in <module>
     18
     19 import tensorflow as tf
---> 20 import deeplabcut.pose_estimation_tensorflow.backbones.efficientnet_builder as eff
     21 from .base import BasePoseNet
     22 from .factory import PoseNetFactory

~\AppData\Local\Continuum\anaconda3\envs\dlc2\lib\site-packages\deeplabcut\pose_estimation_tensorflow\backbones\efficientnet_builder.py in <module>
     18 import tensorflow as tf
     19
---> 20 import deeplabcut.pose_estimation_tensorflow.backbones.efficientnet_model as efficientnet_model
     21 import deeplabcut.pose_estimation_tensorflow.nnets.utils as utils
     22

~\AppData\Local\Continuum\anaconda3\envs\dlc2\lib\site-packages\deeplabcut\pose_estimation_tensorflow\backbones\efficientnet_model.py in <module>
     23 import tensorflow as tf
     24
---> 25 import deeplabcut.pose_estimation_tensorflow.nnets.utils as utils
     26
     27 GlobalParams = collections.namedtuple('GlobalParams', [

AttributeError: module 'deeplabcut' has no attribute 'pose_estimation_tensorflow'
```

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/DeepLabCut/DeepLabCut/blob/master/CODE_OF_CONDUCT.md)

**Comment by MMathisLab** [comment]
@Alex-Denman seems the buidl failed, and/or somehow you are not using the correct version or tensorflow. can you run `import tensorflow.__version__` ? 

can you also include which python / ipython is inthe env, namely, the lines before [1], thanks

**Comment by ByrdOfAFeather** [comment]
I have this issue as well, using tensorflow 2.1.0. 
Happens when trying to import deeplabcut after install.

**Comment by MeysamAmirsardari** [comment]
I have the same issue too, on Ubuntu and using TensorFlow version 2.2.0.
It's my first try after installing DeepLabCut.

**Comment by MeysamAmirsardari** [comment]
I have the same issue on Ubuntu and using TensorFlow version 2.2.0.
it's my first try after installing DeepLabCut.


`>>> import deeplabcut
DLC loaded in light mode; you cannot use any GUI (labeling, relabeling and standalone GUI)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/workspace14/DLC/lib/python3.6/site-packages/deeplabcut/__init__.py", line 46, in <module>
    from deeplabcut.create_project import (
  File "/workspace14/DLC/lib/python3.6/site-packages/deeplabcut/create_project/__init__.py", line 2, in <module>
    from deeplabcut.create_project.demo_data import load_demo_data
  File "/workspace14/DLC/lib/python3.6/site-packages/deeplabcut/create_project/demo_data.py", line 15, in <module>
    from deeplabcut.utils import auxiliaryfunctions
  File "/workspace14/DLC/lib/python3.6/site-packages/deeplabcut/utils/__init__.py", line 1, in <module>
    from deeplabcut.utils.auxfun_multianimal import *
  File "/workspace14/DLC/lib/python3.6/site-packages/deeplabcut/utils/auxfun_multianimal.py", line 20, in <module>
    from deeplabcut.generate_training_dataset import trainingsetmanipulation
  File "/workspace14/DLC/lib/python3.6/site-packages/deeplabcut/generate_training_dataset/__init__.py", line 12, in <module>
    from deeplabcut.generate_training_dataset.trainingsetmanipulation import *
  File "/workspace14/DLC/lib/python3.6/site-packages/deeplabcut/generate_training_dataset/trainingsetmanipulation.py", line 23, in <module>
    from deeplabcut.pose_estimation_tensorflow import training
  File "/workspace14/DLC/lib/python3.6/site-packages/deeplabcut/pose_estimation_tensorflow/__init__.py", line 15, in <module>
    from deeplabcut.pose_estimation_tensorflow.core.train import *
  File "/workspace14/DLC/lib/python3.6/site-packages/deeplabcut/pose_estimation_tensorflow/core/train.py", line 29, in <module>
    from deeplabcut.pose_estimation_tensorflow.nnets import PoseNetFactory
  File "/workspace14/DLC/lib/python3.6/site-packages/deeplabcut/pose_estimation_tensorflow/nnets/__init__.py", line 15, in <module>
    from .efficientnet import PoseEfficientNet
  File "/workspace14/DLC/lib/python3.6/site-packages/deeplabcut/pose_estimation_tensorflow/nnets/efficientnet.py", line 20, in <module>
    import deeplabcut.pose_estimation_tensorflow.backbones.efficientnet_builder as eff
  File "/workspace14/DLC/lib/python3.6/site-packages/deeplabcut/pose_estimation_tensorflow/backbones/efficientnet_builder.py", line 20, in <module>
    import deeplabcut.pose_estimation_tensorflow.backbones.efficientnet_model as efficientnet_model
  File "/workspace14/DLC/lib/python3.6/site-packages/deeplabcut/pose_estimation_tensorflow/backbones/efficientnet_model.py", line 25, in <module>
    import deeplabcut.pose_estimation_tensorflow.nnets.utils as utils
AttributeError: module 'deeplabcut' has no attribute 'pose_estimation_tensorflow'
`

**Comment by MMathisLab** [comment]
@MeysamAmirsardari 3.6 is not supported; please use the conda file. 
@Alex-Denman, you aren't in the DEEPLABCUT env, from your paths I can see you are in `dlc2`

I assure you all it's passing tests (see tests) and works (feel free to open on colab, [here](https://github.com/MMathisLab/DeepLabCut/blob/master/examples/COLAB_maDLC_TrainNetwork_VideoAnalysis.ipynb)), so this is related to your specific install procedure. Please detail how exactly you are installing DLC. 

Here is what we strongly recommend: https://deeplabcut.github.io/DeepLabCut/docs/installation.html

IN short: 

Click [HERE](http://www.mackenziemathislab.org/s/DEEPLABCUT.yaml) to download the conda file!
then in your downloads folder, just run: 

`conda env create -f DEEPLABCUT.yaml`

watch it built, check for errors, and activate based on what it asks ;)

**Comment by ghost** [comment]
@MMathisLab I figured out what was going on, my original issue was erroneous. I mixed up two environments and didn't realize I was in an environment from a previous DLC install with an older version of python. Entirely my mistake. Sorry!
